mcodingbot/utils/search.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from typing import Mapping, TypeVar
+
+from rapidfuzz import fuzz, process
+
+K = TypeVar("K")
+V = TypeVar("V")
+
+
+def fuzzy_search(
+    query: str,
+    choices: Mapping[K, V],
+    *,
+    score_cutoff: int | float | None,
+    limit: int,
```

Comment:
kw-only arguments should have a default value imo.

Just make both of these None, I think?

Reply:
wait lunar just asked you to change that lol. hm

Reply:
okay yeah if you can make this arguments None that would be great.

Reply:
@CircuitSacul leave a suggestion then i will change it